webapp/src/main/webapp/news-details-app/main.js

```diff
@@ -0,0 +1,58 @@
+import './initComponents.js';
+import { newsConstants } from '../js/newsConstants.js';
+
+Vue.use(Vuetify);
+const vuetify = new Vuetify(eXo.env.portal.vuetifyPreset);
+
+// get overridden components if exists
+if (extensionRegistry) {
+  const components = extensionRegistry.loadComponents('NewsDetailApp');
+  if (components && components.length > 0) {
+    components.forEach(cmp => {
+      Vue.component(cmp.componentName, cmp.componentOptions);
+    });
+  }
+}
+
+let newsDetailApp;
```

Comment:
```suggestion
let newsDetailsApp;
```

use newsDetails everywhere